### PR TITLE
PLATUI-286 Makefile to containerise service

### PR DESCRIPTION
**Pull request by JamesLuaHMRC**

### CODE REVIEW

File: docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+set -ex
+
+TOKEN=${TOKEN} npm start
```
Comment (silasl) [inline docker/run.sh]: Does this allow for no token? how are we passing this in?

Comment (JamesLuaHMRC) [reply docker/run.sh]: Yup, it allows for no token, but it should be passed in as an environment variable when you run docker run: """\$JENKINS_HOME/scripts/run_testing_container.py --cluster \${ECS_CLUSTER_NAME} --product ${COMPONENT_RENDERER_SERVICE_NAME} --product-version ${COMPONENT_RENDERER_IMG_VERSION} --proxy-port ${COMPONENT_RENDERER_PORT} --env-vars TOKEN=\${PLATUI_SERVICE_ACCOUNT_GITHUB_TOKEN} --subnet-ids \${CONTAINER_SUBNET_IDS} --sg-ids \$${COMPONENT_RENDERER_SG_IDS_VAR_NAME} --build-tag "\$BUILD_URL\""""

File: docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+ARG NODE_VERSION
+FROM node:${NODE_VERSION}-slim
+CMD . ./run.sh
+WORKDIR /x-govuk-component-renderer
```
Comment (silasl) [inline docker/Dockerfile]: Awesome - shall we rename the repo?

File: docker/run.sh (hunk 1 above)
Comment (JamesLuaHMRC) [reply docker/run.sh]: This is done in the build job - token should be stored by B&D in Build Jenkins - a blank token doesn't seem to cause the service to fail :)

File: docker/Dockerfile (hunk 2 above)
Comment (JamesLuaHMRC) [reply docker/Dockerfile]: Haha, yup, that would be cool!

**Pull request merged**